monero-outreach-docs/translations/es/storyboard/story58.md

```diff
@@ -0,0 +1,67 @@
+# Monero como el Gigante Durmiente
+
+ _"La mentalidad fuerte es aguda y penetrante, rompe la costra de las leyendas y mitos y separa lo que es verdadero de lo falso. El individuo fuerte de espíritu es astuto y discernidor. Posee una mentalidad fuerte, austera, que le proporciona firmeza de propósito y solidez en los compromisos. ¿Quién pondrá en duda que esta fortaleza de espíritu es una de las más grandes necesidades del hombre? Pocas veces encontramos hombres que de buena gana se comprometan a pensar con firmeza y solidez. Existe una tendencia casi universal a buscar las respuestas fáciles y las soluciones para salir del paso. Nada molesta más a la gente que tener que pensar."_  

```

Comment:
This is a fragment of an official translation of the book of Luther Jr. mentioned. I think it's more appropriate to use an official translation since it's a quote and not a line written by the author of this publish.

Reply:
Ok, you have a point.